Source/FakeItEasy/Core/IFakeObjectOptionsBuilder.cs

```diff
@@ -0,0 +1,18 @@
+namespace FakeItEasy.Core
+{
+    using System;
+    using Creation;
+
+    /// <summary>
+    /// Handles global configuration of fake objects by building fake creation options.
+    /// </summary>
+    public interface IFakeObjectOptionsBuilder

```

Comment:
You probably know as much as we do. It seems to be something that's used to build options. Users can supply a new kind when creating a fake scope. I've wondered if this is ever done. Had we reliable feature usage statistics, I'd be asking "Should we drop the `CreateScope` calls that take an `IFakeObjectContainer`?".
I'm especially of this mindset after reading 

> My criteria there was “do I groan anytime a user asks me a question about this feature…” If the answer was “yes”, I killed it.

in [Deleting Code](http://jeremydmiller.com/2016/01/13/deleting-code/).